client/src/components/App/app.jsx

```diff
@@ -74,6 +54,10 @@ class App extends React.Component {
     this.handleOutsideClick=this.handleOutsideClick.bind(this);
   }
 
+  shouldComponentUpdate(nextProps,nextState){
```

Comment:
What does this lifecycle method do in the context of your app?

Reply:
it only rerenders when state or props change